### PR TITLE
DOC-612: toolbar drawer content determination

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-612

Description of Changes:
* Add information on how the editor determines which toolbar buttons to move to the more drawer.

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Comment by tylerkelly13** [comment]
Currently targeting the TinyMCE 5.4 release branch to avoid potential issues during scheduled release, can be updated to target master branch if required.

File: _includes/configuration/toolbar-mode.md [hunk 1]
```diff
@@ -6,6 +6,25 @@
 
 The `toolbar_mode` option is used to extend the toolbar to accommodate the overflowing toolbar buttons. This option is useful for small screens or small editor frames.
 
+The editor will move toolbar buttons to the toolbar drawer based on:
+
+- The defined toolbar groups.
+- The width of the editor.
+
+For example. If the following toolbar configuration is wider than the editor, the toolbar will be split at the group separator (`|`).
```
Comment (metricjs) [inline _includes/configuration/toolbar-mode.md:14]: "toolbar will be split" - is that clear enough? would we want to be explicit and say something like "the items before the separator will appear in the main toolbar and the rest will be moved to the drawer"?

File: _includes/configuration/toolbar.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ tinymce.init({
 });
 ```
 
+> **Note**: The toolbar buttton groups defined in the `toolbar` setting will affect which toolbar buttons will be shown when the toolbar is wider than the editor. For more information, see: [the `toolbar_mode` setting]({{site.baseurl}}/configure/editor-appearance/#toolbar_mode).
```
Comment (metricjs) [inline _includes/configuration/toolbar.md:20]: does it matter that this is specifically if toolbar_mode is floating or sliding?

File: _includes/configuration/toolbar-mode.md [hunk 3]
```diff
@@ -6,6 +6,25 @@
 
 The `toolbar_mode` option is used to extend the toolbar to accommodate the overflowing toolbar buttons. This option is useful for small screens or small editor frames.
 
+The editor will move toolbar buttons to the toolbar drawer based on:
```
Comment (metricjs) [inline _includes/configuration/toolbar-mode.md:9]: Should specify that this applies to floating and sliding modes and not the other two?

File: _includes/configuration/toolbar.md (hunk 2 above)
Comment (lnewson) [reply _includes/configuration/toolbar.md:20]: Also, there's a typo here `buttton` (one too many t's).

File: _includes/configuration/toolbar-mode.md [hunk 4]
```diff
@@ -6,6 +6,25 @@
...
+tinymce.init({
+    selector: 'textarea',
+    toolbar: 'undo redo styleselect bold italic alignleft aligncenter alignright alignjustify | bullist numlist outdent indent'
+});
+```
+
+Two situations to consider when organising the toolbar:
+
+- If the first toolbar button group is wider than the editor, the whole toolbar will be shown in the toolbar drawer.
+- If no toolbar button groups are defined, the `toolbar_mode` will default to [the `wrap` toolbar mode](#wrap).
```
Comment (lnewson) [inline _includes/configuration/toolbar-mode.md:26]: Is this true? If it is, it's definitely a bug as the whole toolbar should appear in the drawer in this case.

Comment (metricjs) [reply _includes/configuration/toolbar-mode.md:26]: Yeah, I was going to ask about this to...

Comment (lnewson) [reply _includes/configuration/toolbar-mode.md:26]: So I checked this on fiddle and it does wrap. That is a bug, as it should be getting placed in a drawer.

Comment (lnewson) [reply _includes/configuration/toolbar-mode.md:26]: Logged TINY-6260 to fix the bug in TinyMCE.

File: _includes/configuration/toolbar-mode.md (hunk 1 above)
Comment (tylerkelly13) [reply _includes/configuration/toolbar-mode.md]: May of been fixed by: https://github.com/tinymce/tinymce-docs/pull/1612/commits/85c06bb5d09d833d450f339a14642ad85c3ad1db

File: _includes/configuration/toolbar.md (hunk 2 above)
Comment (tylerkelly13) [reply _includes/configuration/toolbar.md]: May of been fixed by: https://github.com/tinymce/tinymce-docs/pull/1612/commits/85c06bb5d09d833d450f339a14642ad85c3ad1db

File: _includes/configuration/toolbar-mode.md (hunk 3 above)
Comment (tylerkelly13) [reply _includes/configuration/toolbar-mode.md]: May of been fixed by: https://github.com/tinymce/tinymce-docs/pull/1612/commits/85c06bb5d09d833d450f339a14642ad85c3ad1db

File: _includes/configuration/toolbar-mode.md (hunk 4 above)
Comment (tylerkelly13) [reply _includes/configuration/toolbar-mode.md]: May of been fixed by: https://github.com/tinymce/tinymce-docs/pull/1612/commits/85c06bb5d09d833d450f339a14642ad85c3ad1db

File: _includes/configuration/toolbar-mode.md [hunk 5]
```diff
@@ -6,6 +6,25 @@
...
+```js
+tinymce.init({
+    selector: 'textarea',
+    toolbar: 'undo redo styleselect bold italic alignleft aligncenter alignright alignjustify | bullist numlist outdent indent'
+});
+```
+
+Two situations to consider when organising the toolbar:
+
+- When there is two or more toolbar button groups, the main toolbar will show as many complete, sequential toolbar groups as possible within the width of the editor.
```
Comment (metricjs) [inline _includes/configuration/toolbar-mode.md:25]: is/are?

Comment (metricjs) [reply _includes/configuration/toolbar-mode.md:25]: Should we add "Any remaining toolbar button groups will be moved to the toolbar drawer." or something to be explicit?

Comment (tylerkelly13) [reply _includes/configuration/toolbar-mode.md]: Thanks! fixed: https://github.com/tinymce/tinymce-docs/pull/1612/commits/220ff653fe1673dd4354a87ea688eec2fe464b37

**Pull request merged**